server/map_rpc_server_test.go

```diff
@@ -74,7 +74,7 @@ func TestInitMap(t *testing.T) {
 		wantCode   codes.Code
 	}{
 		{desc: "init new map", getRootErr: storage.ErrTreeNeedsInit, wantInit: true, root: nil, wantCode: codes.OK},
-		{desc: "init new map, no err", getRootErr: nil, wantInit: true, root: nil, wantCode: codes.OK},
```

Comment:
For clarity, this test was wrong before? It looks like it returned no error and a nil root, which never happens in reality?

Reply:
Yeah, never happens. But if it does then it's probably an error, so the test was improved a bit.